### ISSUE TITLE
Algorithm unsupported error occurs on Server-ServerAuthenticatorAttestationResponse-Resp-10

### ISSUE THREAD

**Issue by prvjhamada** [fido-alliance/conformance-tools-issues#226]
## What protocol and version of the protocol are you testing?

FIDO2

## What is your implementation class?

Server

## What is the version of the tool are you using?
v0.9.420

## Issue description
On Server-ServerAuthenticatorAttestationResponse-Resp-10,
in the tests using the hash algorithm SHA384 (e.g. P-3 or P-6) the following error is occured.
```
Error: the string "Error while creating credential: Error: Unsupported hashing algorithm for browser environment" was thrown, throw an Error :)
```

**Comment by herrjemand** [comment]
Thanks for that @prvjhamada. Resolved in v0.9.421

**Comment by herrjemand** [comment]
Resolved in v0.9.421
